about.css

```diff
@@ -0,0 +1,393 @@
+  height: 2px;
+  background-color: #e25240;
+  margin-top: -7px;
+  margin-bottom: 20px;
+  margin-left: 49%;
+}
+
+.partners {
+  display: flex;
+  gap: 20px;
+  justify-content: center;
+  margin-bottom: -50px;
+}
+
+.partners img {
+  width: 10%;
+}
+
+.main-logo {
+  display: none;
+}
+
+.head-links {
+  display: none;
+}
+
+.nav-jsi a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.aboutpg {
+  background-image: url(./images/agric\ summit\ 2.webp);
+  background-repeat: no-repeat;
+  width: 100%;
+  background-size: 243%;
+}
+
+@media screen and (min-width: 720px) {
+  .nav-ul {
+    display: flex;
+    gap: 60px;
+    justify-content: flex-end;
+    margin-right: 10%;
+  }
+
+  .nav-li {
+    list-style-type: none;
+  }
+
+  .news {
+    border: 5px #ec5242 solid;
+    padding: 2px 20px;
+    margin-top: -5px;
+  }
+
+  .fa {
+    display: none;
+  }
+
+  .aboutpg {
+    background-image: url(./images/agric\ summit\ 2.webp);
+    background-repeat: no-repeat;
+    width: 100%;
+    background-size: 155%;
+  }
+
+  .gtg {
+    font-family: "Lato", sans-serif;
+    font-size: 35px;
+    text-align: center;
+  }
+
+  h4 {
+    font-family: 'COCOGOOSE', sans-serif;
+  }
+
+  .topic {
+    color: #ec5242;
+    font-size: 50px;
+    text-align: center;
+    width: 60%;
+    margin-left: 20%;
+    margin-top: -4px;
+    padding-top: 4%;
+    margin-bottom: 80px;
+    font-family: 'COCOGOOSE', sans-serif;
+  }
+
+  .pg {
+    text-align: center;
+    color: black;
+    border: 3px solid #fff;
+    box-shadow: 0 5px 5px rgb(222 222 222 / 82%), 0 5px 5px rgb(222 222 222 / 82%);
+    padding: 20px 20px;
+    font-family: "Lato", sans-serif;
+    font-weight: 400;
+    font-size: 15px;
+    margin-bottom: 40px;
+    margin-left: 27%;
+    width: 40%;
+    background-color: #fff;
+  }
+
+  .details {
+    text-align: center;
+  }
+
+  .d-link {
+    color: #000;
+    font-weight: 900;
+  }
+
+  .agro-theme {
+    text-align: center;
+    font-family: 'COCOGOOSE', sans-serif;
+  }
+
+  .agro-talk {
+    text-align: center;
+  }
+
+  .b-logo {
+    height: auto !important;
+  }
+
+  .past-events {
+    text-align: center;
+  }
+
+  .event-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: row;
+    gap: 70px;
+  }
+
+  .event-d {
+    color: #fff;
+    position: relative;
+  }
+
+  .event-cover {
+    width: 100%;
+    padding: 4rem 0;
+    background-color: rgba(236, 82, 66, 0.7);
+  }
+
+  .event-1 {
+    background-image: url(./images/agric\ summit.jpg);
+    background-position: center;
+    background-size: cover;
+  }
+
+  .event-2 {
+    background-image: url(./images/agric\ summit\ 4.webp);
+    background-position: center;
+    background-size: cover;
+  }
+
+  .event-p {
+    font-size: 14px;
+    font-weight: 900;
+  }
+
+  footer {
+    margin-right: -8px;
+    display: flex;
+    background-color: #272a31;
+    margin-top: 50px;
+    margin-left: -7px;
+    margin-bottom: -10px;
```

Comment:
- [ ] According to your previous reviewer the background color of the footer on the desktop version is the same as the color on the original template design. Please note the text color used is white. Kindly implement these changes to make your project look professional. 👍 😎
- [ ] [See previous review](https://github.com/Danny-imo/Capstone-project/pull/2#discussion_r1252145459)